packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-panel.tsx

```diff
@@ -219,18 +217,31 @@ const PullRequestPanel = ({
   ...routingProps
 }: PullRequestPanelProps) => {
   const { Link } = useRouterContext()
-  const mergeable = useMemo(() => pullReqMetadata?.merge_check_status === MergeCheckStatus.MERGEABLE, [pullReqMetadata])
-  const isClosed = pullReqMetadata?.state === PullRequestState.CLOSED
-  const isOpen = pullReqMetadata?.state === PullRequestState.OPEN
-  //   const isConflict = pullReqMetadata?.merge_check_status === MergeCheckStatus.CONFLICT
-  const isDraft = useMemo(() => pullReqMetadata?.is_draft, [pullReqMetadata?.is_draft])
-  const unchecked = useMemo(
-    () => pullReqMetadata?.merge_check_status === MergeCheckStatus.UNCHECKED && !isClosed,
-    [pullReqMetadata, isClosed]
-  )
-  const checkData = checks || []
   const [notBypassable, setNotBypassable] = useState(false)
   const [mergeButtonValue, setMergeButtonValue] = useState(actions[0].id)
+  const [accordionValues, setAccordionValues] = useState<string[]>([])
+
+  const handleAccordionValuesChange = useCallback((data: string | string[]) => {
+    if (typeof data === 'string') return
+
+    setAccordionValues(data)
+  }, [])
+
+  const { isMergeable, isClosed, isOpen, isDraft, isUnchecked, isRebasable, isShowMoreTooltip } = useMemo(() => {
+    const isClosed = pullReqMetadata?.state === PullRequestState.CLOSED
+    const isOpen = pullReqMetadata?.state === PullRequestState.OPEN
+    const isDraft = !!pullReqMetadata?.is_draft
+
+    return {
+      isMergeable: pullReqMetadata?.merge_check_status === MergeCheckStatus.MERGEABLE,
+      isClosed,
+      isOpen,
+      isDraft,
+      isUnchecked: pullReqMetadata?.merge_check_status === MergeCheckStatus.UNCHECKED && !isClosed,
+      isRebasable: pullReqMetadata?.merge_target_sha !== pullReqMetadata?.merge_base_sha && !pullReqMetadata?.merged,
+      isShowMoreTooltip: isOpen && !isDraft
+    }
+  }, [pullReqMetadata])
```

Comment:
Since nothing in here is complex and it deals exclusively with booleans, it would probably be as fast to not memoize it and cut down on the memo overhead.